### PR TITLE
Feature/vehicle-calldown

### PR DESCRIPTION
**Pull request by zSleepwalker**
One thing lead to another, so it has a bit extra than what the branch name is saying.

### CODE REVIEW

File: UdpHosts/GameServer/Controllers/Character/CombatController.cs [hunk 1]
```diff
@@ -18,9 +25,93 @@ public void FireInputIgnored(INetworkClient client, IPlayer player, ulong entity
...
+         */
+        var weaponProjectileFired = new WeaponProjectileFired
+        {
+            ShortTime = (ushort)fireWeaponProjectile.Time,
+            Aim = fireWeaponProjectile.AimDirection,
+            HaveMoreData = fireWeaponProjectile.HaveUnkVector,
+            MoreData = fireWeaponProjectile.UnkVector
+        };
+
+        client.NetChannels[ChannelType.ReliableGss].SendIAeroControllerKeyframe(fireWeaponProjectile, player.CharacterEntity.EntityId, player.PlayerId);
```
Comment (Xsear) [inline UdpHosts/GameServer/Controllers/Character/CombatController.cs:49]:
```suggestion
        client.NetChannels[ChannelType.ReliableGss].SendIAero(fireWeaponProjectile, player.CharacterEntity.EntityId);
```

File: UdpHosts/GameServer/Controllers/Character/BaseController.cs [hunk 2]
```diff
@@ -658,4 +742,279 @@ public void SetSteamUserId(INetworkClient client, IPlayer player, ulong entityId
...
+    {
+        var mapOpened = new GeographicalReportResponse
+        {
+            Unk1 = 0,
+            Unk2 = new Vector3 { X = 0, Y = 0, Z = 0},
+            Unk3 = 0x00,
+            Data = new GeoReportData[] { }
+        };
+
+        client.NetChannels[ChannelType.ReliableGss].SendIAeroControllerKeyframe(mapOpened, player.CharacterEntity.EntityId, player.PlayerId);
```
Comment (Xsear) [inline UdpHosts/GameServer/Controllers/Character/BaseController.cs:1018]:
```suggestion
        client.NetChannels[ChannelType.ReliableGss].SendIAero(mapOpened, player.CharacterEntity.EntityId);
```

File: UdpHosts/GameServer/Controllers/Character/BaseController.cs [hunk 3]
```diff
@@ -658,4 +742,279 @@ public void SetSteamUserId(INetworkClient client, IPlayer player, ulong entityId
...
+                    Unk3 = "",
+                    Unk4 = 1,
+                    Unk5 = 1471686583,
+                    Unk6 = 1
+                }
+            },
+            Unk2 = 0
+        };
+
+        client.NetChannels[ChannelType.ReliableGss].SendIAeroControllerKeyframe(friendsListResponse, player.CharacterEntity.EntityId, player.PlayerId);
```
Comment (Xsear) [inline UdpHosts/GameServer/Controllers/Character/BaseController.cs:1004]:
```suggestion
        client.NetChannels[ChannelType.ReliableGss].SendIAero(friendsListResponse, player.CharacterEntity.EntityId);
```

File: UdpHosts/GameServer/Controllers/Character/BaseController.cs [hunk 4]
```diff
@@ -658,4 +742,279 @@ public void SetSteamUserId(INetworkClient client, IPlayer player, ulong entityId
...
+    [MessageID((byte)Commands.ResourceLocationInfosRequest)]
+    public void ResourceLocationInfosRequest(INetworkClient client, IPlayer player, ulong entityId, GamePacket packet)
+    {
+        var resourceLocationInfosResponse = new ResourceLocationInfosResponse
+        {
+            Data = new ResourceLocationInfo[] { },
+            Unk = 0x01
+        };
+
+        client.NetChannels[ChannelType.ReliableGss].SendIAeroControllerKeyframe(resourceLocationInfosResponse, player.CharacterEntity.EntityId, player.PlayerId);
```
Comment (Xsear) [inline UdpHosts/GameServer/Controllers/Character/BaseController.cs:972]:
```suggestion
        client.NetChannels[ChannelType.ReliableGss].SendIAero(resourceLocationInfosResponse, player.CharacterEntity.EntityId);
```

File: UdpHosts/GameServer/Controllers/Character/BaseController.cs [hunk 5]
```diff
@@ -636,16 +716,20 @@ public void MovementInput(INetworkClient client, IPlayer player, ulong entityId,
...
+        };
 
-        packet.Unpack<BagInventorySettings>();
+        client.NetChannels[ChannelType.ReliableGss].SendIAeroControllerKeyframe(bagInventoryUpdate, player.CharacterEntity.EntityId, player.PlayerId);
```
Comment (Xsear) [inline UdpHosts/GameServer/Controllers/Character/BaseController.cs:732]:
```suggestion
        client.NetChannels[ChannelType.ReliableGss].SendIAero(bagInventoryUpdate, player.CharacterEntity.EntityId);
```

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thanks 🥳

**Pull request merged**